examples/benchmarks/tablebench/src/azure.ts

```diff
@@ -9,6 +9,7 @@
  */
 
 import { AzureClient, AzureLocalConnectionConfig } from "@fluidframework/azure-client";
+import { IFluidContainer } from "@fluidframework/fluid-static";
```

Comment:
The alternatives to an import:
1. get the type from the call
2. use a helper function

If we going to keep a type, then use a `type` import, which is not a real dependency in the end.

But here, that is not the right type. The correct type is
```typescript
IFluidContainer<{
    initialObjects: {
        tree: typeof SharedTree;
    };
}>
```
(I don't know why IFluidContainer has a default.)

But why do we need this? When I look at this code without any changes the type is properly determined. What is the change and should we be suspicious?

And finally if it is related to the TODO, well, that bug has been fixed. So, I think the workaround can be removed. @clarenceli-msft ?

Reply:
The reason it was failing lint I think is because I accidentally removed "noImplicitAny": true from the tsconfig. I
fixed that and reverted the code changes.